libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -75,12 +75,12 @@
         AztecToolbarClickListener,
         HistoryListener {
 
-    private static final String ARG_PARAM_TITLE = "param_title";
-    private static final String ARG_PARAM_CONTENT = "param_content";
-
-    private static final String KEY_TITLE = "title";
-    private static final String KEY_CONTENT = "content";
-
+    private static final String ATTR_ALIGN_ = "align-";
```

Comment:
That works for me.  I updated the variables in b54214dacc9239924ec5759c517991ce4b19e06a.